photoprism/Photo.py

```diff
@@ -119,37 +117,31 @@ def start_import(self, path="upload", move=False):
             "move": move
         }
         r = requests.post(url, data=json.dumps(data), headers=self.header)
-        result = False
+        
         if r.status_code == 200:
-            result = True
-        return result
+            return True
+        
+        return False
 
     def stop_import(self):
         """Stop an import job"""
         url = f"{self.session.url}/import"
         r = requests.delete(url, headers=self.header)
-        result = False
+
         if r.status_code == 200:
-            result = True
-        return result
+            return True
+        
+        return False
 
-    def raw_call(self, endpoint, type="GET", data=False):
+    def raw_call(self, endpoint, type="GET", data=None):
         """Function to perform a request to the photoprism server"""
 
         url = f"{self.session.url}/{endpoint}"
         if type == "GET":
-            if data:
-                r = requests.get(url, data = json.dumps(data), headers=self.header)
-            else:
-                r = requests.get(url, headers=self.header)
+            return requests.get(url, headers=self.header)
```

Comment:
Seems that `data` is not available in a get request. But you can reuse the request.post logic below